migrations/versions/8558ca0c7f16_auth.py

```diff
@@ -0,0 +1,61 @@
+"""
+auth
+
+Revision ID: 8558ca0c7f16
+Revises: 143432ee0675
+Create Date: 2022-11-15 13:31:52.495417
+"""
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+
+# revision identifiers, used by Alembic.
+revision = '8558ca0c7f16'
+down_revision = '143432ee0675'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        'auth_group',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('name', sa.String(length=64), nullable=False),
+        sa.PrimaryKeyConstraint('id'),
+        sa.UniqueConstraint('name')
+    )
+    op.create_table(
+        'auth_user',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('external_uuid', postgresql.UUID(), nullable=True),
+        sa.Column('name', sa.String(length=64), nullable=True),
```

Comment:
It can be anything. For now, this is bare minimum to be able to login. Will add some more fields like email, full name, SSH key, sync from LDAP, etc. in the future.

I din't want to create one huge MR, so there are several more MRs to come. Keycloak client will be removed in one of the next MR. In the end, there will be only auth code that use public ouath API to verify tokens.

I'm not sure how exactly everything will work yet. This is just a base module on which we can be build other features.

Reply:
Ok, sounds good